test/e2e/kubevirt.go

```diff
@@ -2359,4 +2360,97 @@ chpasswd: { expire: False }
 		)
 	})
 
+	Context("MAC conflicts detection", func() {
+		newTestVMI := func(name, mac string) *kubevirtv1.VirtualMachineInstance {
+			vm := fedoraWithTestToolingVMI(nil, nil, nil, kubevirtv1.NetworkSource{Pod: &kubevirtv1.PodNetwork{}}, "#", "")
+			vm.Name = name
+			vm.Spec.Domain.Devices.Interfaces[0].Bridge = nil
+			vm.Spec.Domain.Devices.Interfaces[0].Binding = &kubevirtv1.PluginBinding{Name: "l2bridge"}
+			vm.Spec.Domain.Devices.Interfaces[0].MacAddress = mac
+			return vm
+		}
+
+		It("should fail when creating second VM with duplicate static IP", func() {
```

Comment:
don't you mean MAC here ? 

```suggestion
		It("should fail when creating second VM with duplicate user requested MAC", func() {
```